gstools/covmodel/tools.py

```diff
@@ -597,14 +605,14 @@ def model_repr(model):  # pragma: no cover
     else:
         std_str = (
             "{0}(dim={1}, var={2:.{p}}, len_scale={3:.{p}}, "
-            "nugget={4:.{p}}, anis={5}, angles={6}{7})".format(
+            "nugget={4:.{p}}{5}{6}{7})".format(
                 model.name,
                 model.dim,
                 model.var,
                 model.len_scale,
                 model.nugget,
-                list_format(model.anis, 3),
-                list_format(model.angles, 3),
+                ani,
+                ang,
```

Comment:
Hmm, not so sure about f-strings here... :laughing:

Reply:
Done. Was worth the effort.